clang/lib/Driver/Driver.cpp

```diff
@@ -6980,3 +6980,37 @@ void driver::applyOverrideOptions(SmallVectorImpl<const char *> &Args,
       ++S;
   }
 }
+
+/// Vectorize at all optimization levels greater than 1 except for -Oz.
+/// For -Oz the loop vectorizer is disabled, while the slp vectorizer is
+/// enabled.
+bool driver::shouldEnableVectorizerAtOLevel(const ArgList &Args,
```

Comment:
We have made some attempts to move shared logic out of `clang/` into `llvm/lib/Frontend/Driver`. Maybe this should go there too?

Reply:
This should be able to go there I think; I'll move it if the RFC makes progress